specification/containerservice/resource-manager/Microsoft.ContainerService/aks/stable/2024-09-01/managedClusters.json

```diff
@@ -490,6 +490,22 @@
               "$ref": "#/definitions/ManagedCluster"
             },
             "description": "The managed cluster to create or update."
+          },
+          {
+            "name": "If-Match",
+            "in": "header",
+            "description": "The request should only proceed if an entity matches this string.",
+            "required": false,
+            "type": "string",
+            "x-ms-client-name": "ifMatch"
+          },
+          {
+            "name": "If-None-Match",
```

Comment:
Update examples to have these values too.

Reply:
Updated examples to have etag value which is part of response body. The examples were only for what we get as response, I don't see any example for headers passed in even for prior existing headers. Here if-match and if-none-match are header values only.